src/fixate/config/__init__.py

```diff
@@ -28,6 +28,13 @@
 LOG_DIRECTORY = Path(platformdirs.user_log_dir("Fixate", False))
 INSTRUMENT_CONFIG_FILE = CONFIG_DIRECTORY / "instruments.json"
 
+try:
```

Comment:
platform.node() and platform.uname() look like a better way to get this type of info without having to deal with cross platform issues of the os module.
https://docs.python.org/3/library/platform.html#platform.node

Reply:
Yep, thanks!